{"name": "test_patch_user", "status": "failed", "statusDetails": {"message": "AssertionError: Expected status code 200, but got 404", "trace": "api_setup = <utils.service_api.ServiceAPINew.ServiceAPI object at 0x000001E01C137B90>, api_url = 'https://companydev.jobeze.com/backend/user-info'\n\n    def test_patch_user(api_setup, api_url):\n        \"\"\"Test the PATCH request for a specific user ID.\"\"\"\n        user_id = 1881  # Replace with the actual user ID you want to test\n        patch_url = f\"{api_url}/users/{user_id}\"  # Use the same user ID for PATCH\n        payload_patch = {\"value\": \"Updated Sample\"}\n        response_patch = api_setup.patch_service_response(patch_url, payload_patch)\n>       api_setup.validate_response(response_patch, expected_status=200)\n\ntestsuites\\api\\test_service_api.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <utils.service_api.ServiceAPINew.ServiceAPI object at 0x000001E01C137B90>, response = <Response [404]>, expected_status = 200\n\n    @allure.step(\"Validating API response\")\n    def validate_response(self, response, expected_status=200):\n        \"\"\"Validate the API response with the expected status.\"\"\"\n        assert response is not None, \"No response received from the API\"\n>       assert response.status_code == expected_status, \\\n            f\"Expected status code {expected_status}, but got {response.status_code}\"\nE       AssertionError: Expected status code 200, but got 404\n\nutils\\service_api\\ServiceAPINew.py:89: AssertionError"}, "description": "Test the PATCH request for a specific user ID.", "steps": [{"name": "Sending PATCH request to API", "status": "passed", "parameters": [{"name": "api_url", "value": "'https://companydev.jobeze.com/backend/user-info/users/1881'"}, {"name": "payload", "value": "{'value': 'Updated Sample'}"}], "start": 1746169215966, "stop": 1746169216749}, {"name": "Validating API response", "status": "failed", "statusDetails": {"message": "AssertionError: Expected status code 200, but got 404\n", "trace": "  File \"C:\\Internal\\automation-framework-master\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Internal\\automation-framework-master\\utils\\service_api\\ServiceAPINew.py\", line 89, in validate_response\n    assert response.status_code == expected_status, \\\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "parameters": [{"name": "response", "value": "<Response [404]>"}, {"name": "expected_status", "value": "200"}], "start": 1746169216750, "stop": 1746169216750}], "attachments": [{"name": "log", "source": "0dbba92e-a119-44c0-8e5b-1f0c53d4287b-attachment.txt", "type": "text/plain"}], "start": 1746169215966, "stop": 1746169216750, "uuid": "8fb3bc83-b13a-4131-9d31-6d9ebd563fbe", "historyId": "ba5c98e5ad40f6e362a9c5efe43d9378", "testCaseId": "ba5c98e5ad40f6e362a9c5efe43d9378", "fullName": "testsuites.api.test_service_api#test_patch_user", "labels": [{"name": "parentSuite", "value": "testsuites.api"}, {"name": "suite", "value": "test_service_api"}, {"name": "host", "value": "DESKTOP-BR6BRT3"}, {"name": "thread", "value": "34176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testsuites.api.test_service_api"}]}